view/frontend/web/js/generic.js

```diff
@@ -155,15 +159,18 @@ define([
             var attributes = $.extend(getCustomerSpecificAttributes(), config.attributes);
 
             getCartSpecificAttributes(function (cartAttributes) {
-                attributes = $.extend(cartAttributes, attributes);
+                if (cartAttributes) {
+                    attributes = $.extend(cartAttributes, attributes);
 
-                if (config.debug) {
-                    console.log(attributes);
+                    if (config.debug) {
```

Comment:
The debug info should also be logged without cart attributes being present. Can you move it outside of the if statement?